templates/static/js/example.js

```diff
@@ -481,7 +481,7 @@
   function fixScroll() {
     const titleId = window.location.hash;
     if (titleId) {
-      const title = document.querySelector(titleId);
+      const title = document.querySelector(`[id="${titleId.substring(1)}"]`);
```

Comment:
Can you add a comment here explaining the purpose for the substring operation?

Also, I'm not sure if this is necessary - your Python change seems to have successfully changed the ID to exclude the status label.

Current [prod](https://vanillaframework.io/docs/base/forms#dense-help-text):
<img width="1108" alt="Screenshot 2025-05-30 at 20 00 07" src="https://github.com/user-attachments/assets/53928a76-ec66-4bb7-bc2c-c5effb2f9530" />

This [PR](https://vanilla-framework-5531.demos.haus/docs/base/forms#dense-help-text):
<img width="1108" alt="Screenshot 2025-05-30 at 20 00 10" src="https://github.com/user-attachments/assets/10600bb9-cdb6-4424-8e2e-e2e5d3ce56b1" />